package.json

```diff
@@ -75,6 +75,7 @@
     "react-dom": "18.2.0",
     "react-icons": "^4.11.0",
     "react-intersection-observer": "^9.5.2",
+    "react-router-dom": "^6.18.0",
```

Comment:
@techyNonso is react-router-dom required for nextjs?

Reply:
it did not work i wanted to use useHistory before

Reply:
oh i see thanks @Codebox124, @techyNonso kindly take a look into this. If its not satifying kindly raise a ticket in next iteration to take a look_